keps/2076-kueuectl/README.md

```diff
@@ -339,7 +340,42 @@ so that the users won't have to remember to switch the command to kubectl.
 * `describe workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
 * `edit workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
 * `patch workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
-* `delete workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
+* `delete clusterqueue|cq|localqueue|lq|resourceflavor|rf`
+
+### Delete Workload
+
+Motivation:
+The pass-through delete workload command doesn't actually delete the Workload but 
+instead requeues it. This can confuse users, as the Workload remains listed even 
+after they attempt to delete it, due to its recreation.
+
+This command will delete the Workload and its corresponding Job(s). If the Workload 
```

Comment:
Does it need to delete the workload object? I think deleting the Job is enough - the workload object will be deleted by Kueue by this code:

Reply:
Ok, but this is a corner case. If the Job exists, no point deleting the workload.

Reply:
hm, or maybe there is a point - to make the operation synchronous - so that the workload is gone if the `kueuectl workload list` command is repeated very fast. If this is the motivation, then please state it in the KEP, otherwise we can just say in the help that the operation is async and may take a while.

Reply:
(prebuild workloads are orphan in multikueue worker clusters until they get admitted)

Reply:
Done